playbooks/files/rax-maas/plugins/maas_common.py

```diff
@@ -537,53 +561,6 @@ def get_ironic_client(token=None, endpoint=None, previous_tries=0):
         return ironic
 
 
-try:
```

Comment:
I think @cloudnull mentioned he wanted to keep this around since magnum was an official project. I'll defer to him though as I'm non-partial. Same goes for additional magnum plugins/files.

Reply:
:+1: while we don't officially support magnum, it is an openstack project and something we have upstream support for. Additionally, as I understand it, we have more magnum in the wild than ironic so I'd hate to remove their monitoring capability.